packages/stream_video_push_notification/lib/src/stream_video_push_notification.dart

```diff
@@ -45,7 +50,33 @@ class StreamVideoPushNotificationManager implements PushNotificationManager {
     required this.androidPushProvider,
     required this.pushParams,
     this.callerCustomizationCallback,
-  }) : _client = client;
+  }) : _client = client {
+    _subscriptions.add(
+      _idCallIncoming,
+      onCallEvent.whereType<ActionCallIncoming>().listen(
+        (_) {
+          _subscriptions.add(
+            _idCallEnded,
+            client.events.on<CoordinatorCallEndedEvent>(
+              (_) {
+                endAllCalls();
+              },
+            ),
+          );
+        },
+      ),
+    );
+
+    _subscriptions.add(
+      _idCallAcceptDecline,
+      onCallEvent.whereType<ActionCallAccept>().map((_) => null).mergeWith(
+          [onCallEvent.whereType<ActionCallDecline>().map((_) => null)]).listen(
+        (_) {
+          _subscriptions.cancel(_idCallEnded);
+        },
+      ),
+    );
+  }
```

Comment:
Hey @Brazol,

I was wondering if we can start listening for these events inside `registerDevice` function 🙂

Reply:
We have to listen to them in the constructor for it to work when the app is closed